client/jetpack-cloud/sections/jetpack-social/promo.jsx

```diff
@@ -0,0 +1,79 @@
+import { format as formatUrl, getUrlParts, getUrlFromParts } from '@automattic/calypso-url';
+import { Gridicon } from '@automattic/components';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+import SocialImage from 'calypso/assets/images/jetpack/rna-image-social.png';
+import DocumentHead from 'calypso/components/data/document-head';
+import JetpackRnaActionCard from 'calypso/components/jetpack/card/jetpack-rna-action-card';
+import Main from 'calypso/components/main';
+import { preventWidows } from 'calypso/lib/formatting';
+import getSiteAdminUrl from 'calypso/state/sites/selectors/get-site-admin-url';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+
+import './promo.scss';
+
+export const Promo = ( { isSocialActive, adminUrl, translate } ) => {
+	const titleHeader = translate( 'Social' );
```

Comment:
Do we want this to be translateable, considering it's the plugin name?

Reply:
Don't know, but it can't hurt! Generally all outputted text is translated